.github/ISSUE_TEMPLATE/support-request.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Support Request
+about: Report an issue faced while using Presto
+title: '[SUPPORT]'
+labels: support
```

Comment:
We might need to create this new label called `support`, which I am introducing here.

Reply:
I am afraid `Support` can be interpreted on the lines of customer support which is not this is intended for.

Reply:
Good idea, thanks @majetideepak